examples/php_lazy_loading/lazy.php

```diff
@@ -0,0 +1,7 @@
+<div id="content">
+    <p>The content below will be lazy loaded!</p>
+    <progress id="target">
+    </progress>
+    <iframe src="slow.php#target" loading=lazy style="height:0;opacity:0"
```

Comment:
How about `width: 0; height: 0; border: 0;`? And add quotes around `lazy` for consistency.

Reply:
I wonder if `width:0` won't break on Firefox (width:0 and height:0 is interpreted as non-visible and should never load)

I don't think it matters, this repo seems abandoned :D

Reply:
No, `width: 0` & `height: 0` is not like `display: none`, it's more like `opacity: 0`.